src/pages/tutor-profile/TutorProfile.js

```diff
@@ -1,5 +1,13 @@
+import Box from '@mui/material/Box'
+import MainInfo from '~/containers/tutor-profile/main-info/MainInfo'
+
 const TutorProfile = () => {
-  return <div>TutorProfile Page Placeholder</div>
+  return (
+    <Box sx={ { maxWidth: '1158px', margin: '0 auto' } }>
```

Comment:
use <Container> to wrap page and give flex 1 to it

Reply:
fixed